src/content/docs/reference/Buffers/texture_format.mdx

```diff
@@ -8,115 +8,138 @@ sidebar:
 
 import { Aside } from '@astrojs/starlight/components';
 
-Color attachments, such as [colortex](/reference/buffers/colortex) and [shadowcolor](/reference/buffers/shadowcolor) buffers, store values in a specific format. The default format is `RGBA` (which equates to `RGBA8` on most systems), however a shader pack can specify the format and precision of each color attachment.
+Color attachments such as [colortex](/reference/buffers/colortex) and [shadowcolor](/reference/buffers/shadowcolor) buffers, as well as [images](/reference/buffers/custom_images/) and [textures](/reference/buffers/custom_textures/), store values in a specific format.
 
-The format for any color attachement can be set using the [\<bufferName\>Format](/reference/constants/buffer_format) directive.
+The default format for color attachments is `RGBA` (which equates to `RGBA8` on most systems). However, a shader pack can specify the format using the [\<bufferName\>Format](/reference/constants/buffer_format) directive.
 
-The available texture formats and their meanings are described below:
+The available texture formats and their meanings are described below.
 
-## Normalized
-
-Normalized buffers can store "floating point" values in a fixed range, between 0.0 and 1.0 (inclusive).  The values are not actually stored as floating point internally, but as an integer value. However, the result is converted to a floating point during read/write to a color attachment using a normalized format, so the shader will read/write floating point values to the attachments, however any values outside of the range will be clamped to fit in the range.
-
-| 8-bit | 16-bit | mixed    |
-| ----- | ------ | -------- |
-| R8    | R16    | RGB5_A1  |
-| RG8   | RG16   | RGB10_A2 |
-| RGB8  | RGB16  |          |
-| RGBA8 | RGBA16 |          |
+> Cells marked with ~strikethrough~ are currently unsupported.
 
-## Signed Normalized
-
-Signed Normalized buffers are identical to normalized buffers, however the range they can represent is expanded to -1 to 1 inclusive, allowing them to store negative values.
+## Normalized
 
-| 8-bit       | 16-bit       |
-| ----------- | ------------ |
-| R8_SNORM    | R16_SNORM    |
-| RG8_SNORM   | RG16_SNORM   |
-| RGB8_SNORM  | RGB16_SNORM  |
-| RGBA8_SNORM | RGBA16_SNORM |
+Normalized buffers store "floating point" values in a fixed range. The values are stored as integers internally, which are converted to/from floating point numbers during read/write. Any values outside the range are clamped to fit in the range.
+
+Detailed explanation: [OpenGL Wiki - Normalized Integer](https://www.khronos.org/opengl/wiki/Normalized_Integer)
+
+### Unsigned Normalized Formats
+
+Unsigned normalized buffers support values from 0.0 to 1.0 (inclusive).
+
+| Image Format | Bit Depth  | Pixel Format | Pixel Type                    | Image | Requirements                                                                                                         |
```

Comment:
Should this be "Texture Format" instead of "Image Format"? Or is Image Format the correct term?

Reply:
> Should this be "Texture Format" instead of "Image Format"? Or is Image Format the correct term?

https://github.com/IrisShaders/DocsPage/pull/331#issuecomment-2383735981

After quickly reading the OpenGL specification and wiki i think image format is the more correct term

The wiki page most of what i wrote is based on is also called "Image Format"